app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
```

Comment:
For different cases we can create different if-else branches

```
if '-f' in argv:
    ...
if '-d' in argv:
    ...
if '-f' in argv and '-d' in argv:
    ...
```

Reply:
Much better to create a function for parsing arguments and after do logic based on parsed args